css-typed-om/Overview.bs

```diff
@@ -336,6 +336,8 @@ interface SimpleLength : LengthValue {
 };
 </pre>
 
+If the provided DOMString or {{LengthValue}} is invalid the relevant function throws a SyntaxError.
```

Comment:
Need to describe why each method can throw.

e.g. add, etc. throw if called on or passed a LengthValue that isn't a SimpleLength or CalcLength.
parse throws if the DOMString isn't a valid representation of a length.
I don't think fromValue, fromDictionary can throw.